### PR TITLE
fix: Update left nav

### PR DESCRIPTION
**Pull request by LisaDelaney**

### CODE REVIEW

File: src/rocm_docs/rocm_docs_theme/flavors/rocm-docs-home/left-side-menu.jinja [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {%
-set main_doc_link = ("ROCm Documentation Home", projects['rocm'])
+set main_doc_link = ("ROCm documentation", projects['rocm'])
```
Comment (samjwu) [inline src/rocm_docs/rocm_docs_theme/flavors/rocm-docs-home/left-side-menu.jinja:2]: This is intended to point to the homepage

eg: https://rocm.docs.amd.com/en/latest/index.html

File: src/rocm_docs/rocm_docs_theme/flavors/rocm/left-side-menu.jinja [hunk 2]
```diff
@@ -1,3 +1,3 @@
 {%
-set main_doc_link = ("ROCm API & tools list", projects['rocm'] ~ "/reference/library-index.html")
+set main_doc_link = ("ROCm docs homepage", projects['rocm'] ~ "index.html")
```
Comment (samjwu) [inline src/rocm_docs/rocm_docs_theme/flavors/rocm/left-side-menu.jinja]: This is intended to point to the list, either API or tools

eg:

projects['rocm'] ~ "reference/api.html"

File: src/rocm_docs/rocm_docs_theme/flavors/rocm-docs-home/left-side-menu.jinja (hunk 1 above)
Comment (LisaDelaney) [reply src/rocm_docs/rocm_docs_theme/flavors/rocm-docs-home/left-side-menu.jinja:2]: yes, my intent is to shorten the name and make it sentence case

File: src/rocm_docs/rocm_docs_theme/flavors/rocm/left-side-menu.jinja (hunk 2 above)
Comment (LisaDelaney) [reply src/rocm_docs/rocm_docs_theme/flavors/rocm/left-side-menu.jinja]: in the latest ROCm docs iteration, I split the APIs and the tools, so thought it best to just link back to the ROCm docs homepage

Comment (LisaDelaney) [inline src/rocm_docs/rocm_docs_theme/flavors/rocm/left-side-menu.jinja]:
```suggestion
set main_doc_link = ("ROCm documentation", projects['rocm'] ~ "index.html")
```

File: src/rocm_docs/rocm_docs_theme/flavors/rocm/left-side-menu.jinja [hunk 3]
```diff
@@ -1,3 +1,3 @@
 {%
-set main_doc_link = ("ROCm API & tools list", projects['rocm'] ~ "/reference/library-index.html")
+set main_doc_link = ("ROCm documentation", projects['rocm'] ~ "index.html")
```
Comment (samjwu) [inline src/rocm_docs/rocm_docs_theme/flavors/rocm/left-side-menu.jinja:2]:
```suggestion
set main_doc_link = ("ROCm documentation", projects['rocm'])
```

**Review by samjwu — APPROVED** [review]

File: src/rocm_docs/rocm_docs_theme/flavors/rocm-docs-home/left-side-menu.jinja (hunk 1 above)
Comment (LisaDelaney) [inline src/rocm_docs/rocm_docs_theme/flavors/rocm-docs-home/left-side-menu.jinja:2]:
```suggestion
set main_doc_link = ("ROCm documentation", projects['rocm'] ~ "index.html")

```

Comment (samjwu) [reply src/rocm_docs/rocm_docs_theme/flavors/rocm-docs-home/left-side-menu.jinja:2]: This has the same meaning

**Pull request merged**